README.md

```diff
@@ -39,6 +39,8 @@ var rokka = require('rokka')({
   apiVersion: <number>, // default: 1
   renderHost: '<url>',  // default: https://{organization}.rokka.io
   debug: true           // default: false
+  retryOptions: {}      // default: {retries: 10, minTimeout: 1000,
```

Comment:
mmh, the problem with "transport" is that minTimeout and maxTimeout may not be clear, that it's for "retries" and not for general requests.

We can discuss it personally, what's the best approach when you have a minute